decentraland/unity-renderer
# Fix: UI Broken Gradient Emotes

…n't had inner color.

## What does this PR change?



This PR fixes a bug where the gradient of the emotes wheel is too thick and doesn't have an inner color.

## How to test the changes?



1. Go to: https://play.decentraland.zone/?renderer-branch=fix/UI-Broken-Gradient-Emotes
2. Open the emote wheel.
3. Verify the gradient is working as intended

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
